README.md

```diff
@@ -89,10 +89,22 @@ You can create an user using the web interface at [`localhost:5000`](http://loca
 paster --plugin=ckan sysadmin -c ckan/development.ini add <USERNAME>
 ```
 
-You can repeat the last command at any time to start developing again.
+The way Giftless is designed makes it required to add an extra configuration in `docker/giftless.yaml` **if running with local storage**. Add the `- giftless.auth.allow_anon:read_only` bit (there is no space between after `:` — it is a string, not a key/value pair) at the bottom of `AUTH_PROVIDERS`, making your file look like this:
```

Comment:
Why not make it the default config shipped in the project? After all this is a development config, not for production. Plus I would remove or edit this explanation to be much shorter.

Reply:
>  After all this is a development config, not for production.

Great point. Gonna change that — way better I think!